csharp-advanced-topics/SpanInCsharp/CryptographyDotnet/CryptographyDotnet/RSATests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CryptographyDotnet
+{
+    [TestClass]
+    public class RSATests
+    {
+        [TestMethod]
+        public void WhenUsingRSA_ThenDataIsEncrypted()
+        {
+            var dataStr = "This is corporate research! Dont read me!";
+            var data = Encoding.ASCII.GetBytes(dataStr);
+
+            GenerateKeys(2048, out var publicKey, out var privateKey);
```

Comment:
Can you put the `2048` into a meaningfully named variable? Just to make sure our  readers know what this variable is storing, please do that on all places :)